pkgs/by-name/cl/clickhouse/generic.nix

```diff
@@ -154,8 +156,12 @@ llvmPackages_19.stdenv.mkDerivation (finalAttrs: {
     substituteInPlace $out/etc/clickhouse-server/config.xml \
       --replace-fail "<errorlog>/var/log/clickhouse-server/clickhouse-server.err.log</errorlog>" "<console>1</console>" \
       --replace-fail "<level>trace</level>" "<level>warning</level>"
+    remove-references-to -t ${llvmPackages_19.stdenv.cc} $out/bin/clickhouse
```

Comment:
I initially wanted to use `finalAttrs.stdenv.cc` here but `stdenv` doesn't exist there, would a `let`-binding for `llvmStdenv` or similar make sense, so there isn't need to repeat `llvmPackages_19.stdenv` 3 times?

Reply:
Yeah just bind it